src/main/java/com/puppycrawl/tools/checkstyle/api/AutomaticBean.java

```diff
@@ -64,6 +65,7 @@
      *
      * @return a configured BeanUtilsBean
      */
+    // -@cs[ExecutableStatementCount] No other way around adding these beans.
```

Comment:
when you say `custom` you mean any that I have been adding recently?
So that would only be: Pattern, SeverityLevel, Scope, URI